javascript/node/selenium-webdriver/README.md

```diff
@@ -217,7 +217,7 @@ under the License.
 
 [LTS]: https://github.com/nodejs/LTS
 [PATH]: http://en.wikipedia.org/wiki/PATH_%28variable%29
-[api]: http://seleniumhq.github.io/selenium/docs/api/javascript/module/selenium-webdriver/
+[api]: https://www.selenium.dev/documentation/webdriver
```

Comment:
```suggestion
[api]: https://www.selenium.dev/selenium/docs/api/javascript/
```

Reply:
but also the Javascript documentation appears to be broken; none of the links on that page work...

Reply:
Which link does not work? I tried a few and they worked.

Reply:
whoa, looks like I somehow had the old page cached. Very weird. Never mind.

Reply:
Ah yes, that happens often with GitHub pages.

Reply:
This is the URL I was thinking to use as well initially: https://www.selenium.dev/selenium/docs/api/javascript/

However, that URL is pointing back to the README file (just different view/interface) so I thought it might be better to use: https://www.selenium.dev/documentation/webdriver

Anyways, I made a new commit that points to the URL you suggested.